app/posts/apply-for-teacher-training/2023-03-24-merging-the-personal-statement.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Merging the personal statement 
+description: We merged the two personal statement sections into one and improved the guidance around what we say to candidates to help them write a good personal statement
+date: 2023-03-24
+tags:
+  - personal statement
+#ogImage:
+#src: /apply-for-teacher-training/merged-personal-statement.png
+#alt: "Screenshot of the personal statement page, with guidance text and a text box."
+
+---
+
+Previously, the personal statement had two sections on the application form: ‘Why you want to teach’ (600 words) and ‘Your suitability to teach a subject or age group’ (400 words). 
+
+We merged these personal statement sections into one, and improved the guidance around what we say to candidates to help them write a good personal statement. 
+
+We tested this design and improved the content shown to candidates after conducting research. We released this feature, via a feature flag, on 20th March 2023.
+
+## The issue
+
+Candidates found it difficult splitting up their experience and knowledge they have into separate questions and found there was a lot of overlap between the two sections. Primary applicants had particular difficulty in answering separate questions, because ‘why you want to be a teacher’ and ‘why you want to teach Primary’ are very similar. 
+
+We wanted to test whether our new guidance aligned with what candidates expect to write about, and whether it was easy to read and comprehend the guidance. 
+
+We also wanted to learn more about how candidates would approach writing a personal statement, and what worries candidates about doing so. This will help determine what support we can give potential candidates in writing their personal statement. 
+
+
+‘Personal statement’ is one of the highest reasons for rejection used by providers. In merging the personal statement sections into one, there was scope for us to refine our guidance to help candidates improve the quality of their personal statement. 
```

Comment:
We could add a stat here? seems like Personal statement accounts for at least 18% of rejections? (possibly higher, as so many rejections use "other")